### DISCUSSION TITLE
STT for Russian language

### DISCUSSION THREAD

**Discussion by annaklyueva** [coqui-ai/STT#2163] [Q&A] [answered]
Hello!

Could you tell me if there is a STT model for Russian language? Or is there a guidelines how to train (fine-tune) STT model for Russian language?

All the best, 
Anna

**Comment by domcross** [comment, marked as answer]
There is one in the [model zoo](https://coqui.ai/models): https://coqui.ai/russian/jemeyer/v0.1.0 
WER 32%/CER 22% doesn't sound that it is usable for serious tasks, though.

Not compatible with Coqui-STT but [Alphacephei VOSK](https://alphacephei.com/vosk/models) has some models, too.
